auth/server/src/main/java/com/seagates3/dao/ldap/LDAPUtils.java

```diff
@@ -353,11 +373,15 @@ static void bind(String dn, String password) throws LDAPException {
                        ldapException.getCause() + ". Message: " +
                        ldapException.getMessage());
           LOGGER.debug("Stacktrace: " + ldapException);
-          IEMUtil.log(
-              IEMUtil.Level.ERROR, IEMUtil.LDAP_EX,
-              "An error occurred while establishing ldap connection. For " +
-                  "more " + "information, please refer Troubleshooting Guide.",
-              String.format("\"cause\": \"%s\"", ldapException.getCause()));
+        //   IEMUtil.log(
```

Comment:
Migration is not supposed to be part of this patch.

Reply:
commented this new code and reverts the original code for IEM alert.